routes/api.php

```diff
@@ -16,5 +16,7 @@
 */
 
 Route::middleware('auth:api')->get('/user', function (Request $request) {
-    return $request->user();
+    return $request->user;
 });
+
+Route::get('articles', 'ListArticlesController');
```

Comment:
Suggerimento: puoi usare il [fqcn](https://www.php.net/manual/en/language.namespaces.rules.php) importando la classe.

In questo modo se utilizzi un IDE e rinomini la classe o il namespace, automaticamente ti cambiera anche le rotte :) 
```suggestion
Route::get('articles', ListArticlesController::class);
```